src/main/java/dan200/computercraft/shared/peripheral/modem/wired/BlockCable.java

```diff
@@ -181,17 +181,19 @@ public FluidState getFluidState(@Nonnull BlockState state) {
     //            : new ItemStack( ComputerCraftRegistry.ModItems.CABLE.get() );
     //    }
 
-    @Override
-    @Deprecated
-    public boolean canPlaceAt(BlockState state, @Nonnull WorldView world, @Nonnull BlockPos pos) {
-        Direction facing = state.get(MODEM)
-                                .getFacing();
-        if (facing == null) {
-            return true;
-        }
-
-        return sideCoversSmallSquare(world, pos.offset(facing), facing.getOpposite());
-    }
+    // Commenting override to allow cable modems to be placed on chests, so that chests can be generic inventory peripherals.
```

Comment:
This is literally what the full-sized modems exist for! If you do this, they can be placed on torches and stuff, which just looks really janky.

Reply:
Oh yeah whoops that would look janky. I must've misremembered putting cable modems on chests in the forge version.